test/unit/sz.c

```diff
@@ -0,0 +1,55 @@
+#include "test/jemalloc_test.h"
+
+static size_t
+reg_size_compute(int lg_base, int lg_delta, int ndelta) {
+    return (ZU(1) << lg_base) + (ZU(ndelta) << lg_delta);
+}
```

Comment:
Don't copy-paste the code, put this function in the header file and include it here.